CHANGELOG.md

```diff
@@ -1,20 +1,35 @@
-OSV-Scanner v2 is coming soon! The next release will start with version `v2.0.0-alpha1`.
+# v2.0.0-beta1
 
-Here's a peek at some of the exciting upcoming features:
+This beta release of OSV-Scanner introduces significant enhancements, including a refactored dependency extraction capabilities, container image scanning, and guided remediation for Maven. 
 
-- Standalone container image scanning support.
-  - Including support for Alpine and Debian images.
-- Refactored internals to use [`osv-scalibr`](https://github.com/google/osv-scalibr) library for better extraction capabilities.
-- HTML output format for clearer vulnerability results.
-- More control over output format and logging.
-- ...and more!
+This beta release does *not* introduce any breaking changes and the beta period is expected to last approximately one month. However, as this is a beta release, the API may be subject to any change.
 
-Importantly, the CLI interface of osv-scanner will be maintained with minimal breaking changes.
-Most breaking changes will only be in the API. More details in the upcoming alpha release.
+We encourage you to try out these new features and provide feedback as we work towards the v2.0.0 final release.
 
----
+### Enhanced Dependency Extraction with `osv-scalibr`
+
+The core dependency extraction logic has been rewritten to leverage the [`osv-scalibr`](https://github.com/google/osv-scalibr) library. We now support the following additional lockfile formats:
```

Comment:
and also artifact formats? 

Also mention that more will come in future beta releases and with the final 2.0.0 release.